config/kubernetes-sigs/sig-service-catalog/teams.yaml

```diff
@@ -1,10 +1,4 @@
 teams:
-  mcs-api-admins:
-    description: Admin access to the mcs-api repo
-    members:
-    - JeremyOT
-    - pmorie
-    privacy: closed
```

Comment:
Good catch! Can you move this to config/kuberntes-sigs/sig-multicluster/teams.yaml?